Blaze/ByteString/Builder/Internal/Write.hs

```diff
@@ -148,9 +145,6 @@ instance Semigroup Write where
   {-# INLINE (<>) #-}
```

Comment:
same here, this instance can be defined unconditionally now

Reply:
done!